lib/plottr_components/src/components/outline/CardView.js

```diff
@@ -216,6 +216,12 @@ const CardViewConnector = (connector) => {
       return <div className="divider" />
     }
 
+    UNSAFE_componentWillReceiveProps(nextProps, nextState) {
```

Comment:
Could we please try to avoid deprecated lifecycle hooks?
We could achieve the same effect if we used `componentDidUpdate` instead.
Alternatively, I think it would be worth the effort to refactor this to a function component and use a hook.